opensearch-jobs.yml

```diff
@@ -138,6 +138,64 @@ instance_groups:
 #########################################################
 #2nd deploy group - opensearch_data, opensearch_dashboards, ingestors
 #########################################################
+- name: archiver
+  instances: 1
+  jobs:
+  - name: bpm
+    release: bpm
+  - consumes: *consumes-opensearch-manager
+    name: opensearch
+    properties:
+      opensearch:
+        jvm_options:
+        - -Dlog4j2.formatMsgNoLookups=true
+        migrate_data_path: true
+    release: opensearch
+  - name: archiver_syslog
+    properties:
+      logstash:
+        queue:
+          max_bytes: 30gb
+      logstash_archiver:
+        files: 16384
+      logstash_ingestor:
+        outputs:
+        - options:
+            bucket: logs-opensearch-cf-development
+            encoding: gzip
+            prefix: '%{+yyyy/MM/dd/HH/mm}'
+            region: us-gov-west-1
+            server_side_encryption: true
+            time_file: 5
+            validate_credentials_on_root_bucket: false
+          plugin: s3
+    release: opensearch
+  - name: ingestor_cloudfoundry-firehose
+    properties:
+      cloudfoundry:
+        api_endpoint: https://api.dev.us-gov-west-1.aws-us-gov.cloud.gov
+        firehose_cc_pull_interval: 300s
+        firehose_client_id: logsearch_firehose_ingestor
+        firehose_client_secret: doh3eeh7epu3deeviek7ohy7oj7aa9ei
```

Comment:
It seems like this shouldn't be published in the code, since it's a secret value. And I don't see this in the logsearch pipeline?

Reply:
We might want to reset this value for Logsearch as well or whatever deployment we got it from